village/src/components/SmurfForm.js

```diff
@@ -10,15 +11,24 @@ class SmurfForm extends Component {
     };
   }
 
+
   addSmurf = event => {
     event.preventDefault();
     // add code to create the smurf using the api
-
+    axios
+      .post("http://localhost:3333/smurfs", {
+        name: this.state.name,
+        age: this.state.age,
+        height: this.state.height
+      })
+      .then(response => console.log(response))
+      .catch(error => console.log(error));
     this.setState({
       name: '',
       age: '',
       height: ''
     });
+    window.location.reload();
```

Comment:
Another way to make the page update without using .reload would be using setState to update you state again.